manifest.toml

```diff
@@ -3,7 +3,7 @@ packaging_format = 2
 id = "bonfire"
 name = "Bonfire"
 
-version = "0.9.8-beta.26~ynh2"
+version = "1.0.0-rc.2.33~ynh1"
```

Comment:
The version syntax isn't following Yunohost rules, so it won't work.
I don't remember precisely why it fails here.